### PR TITLE
Chain minidom with urllib.request.urlopen

### PR DESCRIPTION
**Pull request by JasonLG1979**
Add minidom to "with" so it's unlinked as soon as it's not needed.

### CODE REVIEW

File: pithos/pandora/pandora.py [hunk 1]
```diff
@@ -420,9 +420,9 @@ def title(self):
                 self._title = self.songName
             else:
                 try:
-                    with urllib.request.urlopen(self.songExplorerUrl) as xml_data:
-                        dom = minidom.parseString(xml_data.read())
-                    attr_value = dom.getElementsByTagName('songExplorer')[0].attributes['songTitle'].value
+                    with urllib.request.urlopen(self.songExplorerUrl) as xml_data, \
+                    minidom.parseString(xml_data.read()) as dom:
```
Comment (TingPing) [inline pithos/pandora/pandora.py]: __[flake8]__
* [E122] continuation line missing indentation or outdented

Comment (JasonLG1979) [inline pithos/pandora/pandora.py]: Ok @TingPing your auto PEP8 checker can't decide? If I indent this it's over intended of I don't it's missing the indentation?

To me it looks better intended but whatever works...

File: pithos/pandora/pandora.py [hunk 2]
```diff
@@ -420,9 +420,9 @@ def title(self):
                 self._title = self.songName
             else:
                 try:
-                    with urllib.request.urlopen(self.songExplorerUrl) as xml_data:
-                        dom = minidom.parseString(xml_data.read())
-                    attr_value = dom.getElementsByTagName('songExplorer')[0].attributes['songTitle'].value
+                    with urllib.request.urlopen(self.songExplorerUrl) as xml_data, \
+                        minidom.parseString(xml_data.read()) as dom:
```
Comment (TingPing) [inline pithos/pandora/pandora.py]: __[flake8]__
* [E125] continuation line with same indent as next logical line

**Comment by TingPing** [comment]
@JasonLG1979 Hope you don't mind that bot too much, it just gets the trivial things out of the way for reviews.

**Comment by JasonLG1979** [comment]
it was just not having that line continuation no matter how I did it,lol!!!

**Comment by TingPing** [comment]
heh, you can also run `flake8` locally to test things.

**Comment by JasonLG1979** [comment]
right on

**Pull request merged**